example/README.md

```diff
@@ -80,3 +80,17 @@ Build the FluentD docker image provided in our example and upload it to your rep
 3. Deploy `fluentd.yaml`
 
    ``kubectl apply -f fluentd.yaml``
+
+## Sending logs to different Dynatrace environments
+
+With the introduction of Cloud Native Full Stack injection [example](https://github.com/Dynatrace/dynatrace-operator/blob/master/config/samples/cloudNativeFullStack.yaml) it is possible to send metrics and traces to more than one Dynatrace environment. The same thing can be done for logs using fluentd. The following example serves as a potential reference implementation. Results may vary.
+
+Adapt the "option 2" instructions above as follows:
+
+1. Create a second API token, from a second Dynatrace environment. Create a second kubernetes for it as well (ex: `log-ingest2`)
```

Comment:
A whole second kubernetes cluster? Isn't this about sending to multiple environments from a single cluster?